### ISSUE TITLE
InvalidOperationException(SqlException) on SqlBulkCopy

### ISSUE THREAD

**Issue by hlkitty** [dotnet/SqlClient#221]
when I insert some rows into table named "User" with SqlBulkCopy,I got a Exception 

> "System.InvalidOperationException:“无法访问目标表“User”。""(The target table "User" could not be accessed.)

and the inner exception is 

> "SqlException: 关键字 'User' 附近有语法错误。"(There is a syntax error near the keyword 'User'.)

**Comment by cheenamalhotra** [comment]
I'm looking towards a potential fix in NetCore here: https://github.com/dotnet/SqlClient/blob/e1e3be4b564eea6023ba5b1ccd00917adf3b37e9/src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs#L535
and in NetFx here:
https://github.com/dotnet/SqlClient/blob/e1e3be4b564eea6023ba5b1ccd00917adf3b37e9/src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs#L630
to change to:
```csharp
updateBulkCommandText.AppendFormat("insert bulk {0} (", "[" + this.DestinationTableName + "]");
```
Somewhere on these lines. I'll give it a try and create a PR soon.

**Comment by Wraith2** [comment]
If you want to escape it you probably want to use make `SqlCommand.ParseAndQuoteIdentifier` internal and then call it, it uses the name parser to sort out multi dotted identifiers and avoid accidental double escaping.

**Comment by v-kaywon** [comment]
Attached is the nupkg with the fix from PR #246. Please verify if the fix works for you.
[Microsoft.Data.SqlClient.1.1.0-dev.zip](https://github.com/dotnet/SqlClient/files/3699999/Microsoft.Data.SqlClient.1.1.0-dev.zip)

**Comment by hlkitty** [comment]
> @hlkitty Attached is the nupkg with the fix from PR #246. Please verify if the fix works for you.
> [Microsoft.Data.SqlClient.1.1.0-dev.zip](https://github.com/dotnet/SqlClient/files/3699999/Microsoft.Data.SqlClient.1.1.0-dev.zip)

@v-kaywon I download this and replaced with the nupkg,and it seems to be working well
